Jenkinsfile

```diff
@@ -7,21 +7,28 @@ def driver_tests = []
 def pkg_tests = []
 def periph_tests = []
 def other_tests = []
-def unittests = []
+def unittests = [];
```

Comment:
ah, thanks for the hint. Still doing some tests here. Will ping you when I'm ready (: